plugins/tinymcespellchecker.md

```diff
@@ -112,85 +117,99 @@ tinymce.init({
 
 {% include configuration/languages.md %}
 
-### `spellchecker_dialog`
+### `spellchecker_languages`
 
-This option specifies the primary spell checking mode. 
+This option specifies the spellchecker languages that are available to the user, provided as a comma delimited string. The specified languages will be available for the spellchecker when using Spell Checker Pro.
 
-- When set to `true`, the [`spellcheck` toolbar button](#toolbarbuttons) will open a dialog that will step the user through each potential spelling error in the document. 
-- When set to `false`, the [`spellcheck` toolbar button](#toolbarbuttons) will enable or disable as-you-type spell checking and the [`spellcheckdialog` toolbar button](#toolbarbuttons) will open the spell checker dialog.
+**Type:** comma-separated `String`
+
+**Default Value:**
+````
+'US English=en_us,UK English=en_gb,Danish=da,Dutch=nl,Finnish=fi,French=fr,German=de,Italian=it,Norwegian=nb,Brazilian Portuguese=pt,Iberian Portuguese=pt_PT,Spanish=es,Swedish=sv'
```

Comment:
You might want to mention the list of languages supported by the spellchecker service as it's pretty limited.

Reply:
Added:
> For a list of available languages, see: [Supported languages](#supportedlanguages).